### DISCUSSION TITLE
A global timekeeping problem postponed by global warming

### DISCUSSION THREAD

**Discussion by HanDeBruijn** [a-cosmology-group/acg#208] [Astrophysics and Physics]
### [Abstract](https://www.nature.com/articles/s41586-024-07170-0)

The historical association of time with the rotation of Earth has meant that Coordinated Universal Time (UTC) closely follows this rotation. Because the rotation rate is not constant, UTC contains discontinuities (leap seconds), which complicates its use in computer networks. Since 1972, all UTC discontinuities have required that a leap second be added. Here we show that increased melting of ice in Greenland and Antarctica, measured by satellite gravity, has decreased the angular velocity of Earth more rapidly than before. Removing this effect from the observed angular velocity shows that since 1972, the angular velocity of the liquid core of Earth has been decreasing at a constant rate that has steadily increased the angular velocity of the rest of the Earth. Extrapolating the trends for the core and other relevant phenomena to predict future Earth orientation shows that UTC as now defined will require a negative discontinuity by 2029. This will pose an unprecedented problem for computer network timing and may require changes in UTC to be made earlier than is planned. If polar ice melting had not recently accelerated, this problem would occur 3 years earlier: global warming is already affecting global timekeeping.

Related: [Earth Just Had Its Shortest Day on Record, Thanks to a 'Wobble'](https://www.sciencealert.com/earth-just-set-a-new-record-for-the-shortest-day-all-thanks-to-a-wobble)

**Comment by HanDeBruijn** [comment]
There are two things here: (1) global warming, melting of ice and rise of the sea level, (2) something that causes The Earth's Rotation Retardation.

So what we have is a question of cause and effect: is (1) the cause of (2) / is (2) an effect of (1) ?

Referring to the book [Origin of Inertia](http://www.alternatievewiskunde.nl/cosmology/ACG_refs/96836893X-rev.pdf) by Amitabha Ghosh, we read on page 83 the following. _Considerable work has been done on the subject_ of the earth's rotation retardation _, and the publications are too many to be exhaustively listed here._ Okay, can we conclude therefore that the above theory by Duncan Agnew is only one of those many?

The work by Amitabha anyway shows that there is not a shred of evidence that there would be _A global timekeeping problem postponed by global warming_. My own attempts concerning The Earth's Rotation Retardation have been published [in this forum](https://github.com/orgs/a-cosmology-group/discussions/198#discussioncomment-8307431) and [on my website](http://www.alternatievewiskunde.nl/cosmology/basis.htm#ERR) and lead to the same conclusion. The **Update** in [his reply](https://github.com/orgs/a-cosmology-group/discussions/153#discussioncomment-8585139) is relevant too.

**Reply by RedshiftDrift** [reply]
External forces cause variations of the Earth's rotation.  Some of those are the distribution of snow, ice, and water (tides, ocean currents), and dominant winds (all that dependent on global weather), and also land masses, internal motion of the magma, the moon, etc.

Ghosh and @HanDeBruijn ***incorrectly conclude*** that because we don't know how to model all these forces, there is no evidence for their influence on the Earth's rotation.  This is a logical error from an improper understanding of [underdetermination](https://en.wikipedia.org/wiki/Underdetermination) and despite plenty of evidence that the earth rotation is an oscillator that interacts with many external forces.  Both Ghosh and @HanDeBruijn dismiss evidence in an attempt to promote their pet theory.

**Reply by HanDeBruijn** [reply]
@RedshiftDrift  writes:
>External forces cause variations of the Earth's rotation.
>the earth rotation is an oscillator that interacts with many external forces.

Yes, we all agree about that.
>Both Ghosh and @HanDeBruijn dismiss evidence in an attempt to promote their pet theory.

No, we don't. **You** are talking about _short term effects_. **We** are talking about a _long term effect_.

If you ask me (but you don't have to): time measurement remains a can of worms. _This page was last edited on 28 March 2024, at 00:03 (UTC)_; we are talking about [Ephemeris time](https://en.wikipedia.org/wiki/Ephemeris_time). Quote mining: _The difference between ET and UT is called_ [&Delta;T](https://en.wikipedia.org/wiki/%CE%94T_(timekeeping)); _it **changes irregularly**, but the **long-term trend** is parabolic, decreasing from ancient times until the nineteenth century, and increasing since then at a rate corresponding to an increase in the solar day length of 1.7 ms per century (see_ [leap seconds](https://en.wikipedia.org/wiki/Leap_second)). Okay, and in the Wikipedia reference about &Delta;T it is read that: _The value of &Delta;T for the start of 1902 was approximately zero; for 2002 it was about 64 seconds. So **Earth's rotations** over that century took about 64 seconds **longer than** would be required for **days of atomic time**._ Indeed, calculation confirms that $365\times 100\times 1.7\times 10^{-3} = 62.05 \approx 64$ seconds.
My ["pet" theory](http://www.alternatievewiskunde.nl/cosmology/basis.htm#ERR) gives orbital days / century that are 35 seconds **longer** that atomic days / century. It predicts that leap seconds will always be positive and never become negative. I think it explains at least part of The Earth's Rotation Retardation

**Reply by HanDeBruijn** [reply]
A(nother) possible cause of [The Earth's Rotation Retardation](http://www.alternatievewiskunde.nl/cosmology/basis.htm#ERR) - mentioned by @RedshiftDrift  - is [Tidal acceleration](https://en.wikipedia.org/wiki/Tidal_acceleration) (Wikipedia), resulting in _Earth's loss of angular momentum and the consequent increase in length of day_.
However, similar results are obtained with the [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs), as has been demonstrated for the [leap second](https://github.com/orgs/a-cosmology-group/discussions/153#discussioncomment-8585139). Cannot resist to throw more math at it: $A=$ age (of the earth), $t=$ atomic time, $\tau=$ orbital time.

$$
1+\tau/A=e^{t/A} \quad \Longrightarrow \quad 1+\tau/A \approx 1+t/A + (t/A)^2/2 \quad \Longrightarrow \quad \Delta T = \tau-t \approx \frac{t^2}{2A}
$$

As mentioned in [NASA - Delta T](https://eclipse.gsfc.nasa.gov/SEcat5/deltat.html) an important detail is that _the value of &Delta;T can be extrapolated from measured values using the long-term mean parabolic trend_. As has been shown two times now, that **parabolic trend** ($\tau^2$ or $t^2$ respectively) is covered indeed by our theory.
Furthermore, the above [Tidal acceleration](https://en.wikipedia.org/wiki/Tidal_acceleration) _moves the Moon to a higher orbit._ And this effect can be [calculated too](http://www.alternatievewiskunde.nl/cosmology/trubble.htm#feit) (roughly) by the Variable Mass Theory. The mathematics is, with $L=$ length in measuring equipment, $r=$ distance in empty space, $m=$ Variable Mass, $H_0=$ Hubble parameter, $t=$ (atomic) time:

$$
\frac{L_0}{L} = \frac{r}{r_0} = \frac{m}{m_0} = e^{H_0 t} \quad \Longrightarrow \quad \frac{\Delta r}{r_0} = \frac{r}{r_0}-1 \approx H_0 t \quad \Longrightarrow \quad \Delta r = H_0 r_0 t
$$

```
# 1 megaParsec
Mpc := 3.08567758*10^22;
# Hubble parameter
hubble := 73.4*1000/Mpc;
# Year in seconds
year := 31556926;
# Earth-Moon distance (m)
EM := 384.4*10^6;
# Drift away within 1 year
dEM := hubble*EM*year;

                     0.02885516655 # 2.89 centimeters
```
According to [NASA - Secular Acceleration of the Moon](https://eclipse.gsfc.nasa.gov/SEcat5/secular.html): _the Moon's mean distance from Earth is increasing by 3.8 cm per year_. One may find that our value of 2.89 cm is not too far off.

Much to my (pleasant) surprise I found the following very concise [article](http://www.alternatievewiskunde.nl/cosmology/ACG_refs/1970AJ_____75__657V.pdf) by Thomas C. van Flandern (1970): [The Secular Acceleration of the Moon](https://adsabs.harvard.edu/full/1970AJ.....75..657V). Here is an interesting quote from the abstract: _Possible causes of this residual acceleration include: (i) additional tidal acceleration; (ii) long-period deficiencies in the lunar theory; (iii) secular variation in length of the Atomic second relative to the Ephemeris_ [= Orbital] _second_. Also take notice again of the _parabolic trends_ for $T^2$ (= centuries squared) everywhere in the paper.

**Reply by ExpEarth** [reply]
It seems you have done a fair bit of work on this, but somehow you're not exactly nailing it.  If the Earth's mass were increasing, that would tend to increase the Moon's orbit to shrink, not expand.  This is because the gravitational force on the Moon would be increased.  I don't see where this is included in your analysis.

**Reply by HanDeBruijn** [reply]
@ExpEarth writes:
>If the Earth's mass were increasing, that would tend to increase the Moon's orbit to shrink

That would be alright if mass would be the only thing that is changing with our Variable Mass Theory (VMT). However, another immediate consequence is (intrinsic) redshift with age. And therefore there must be blueshift in the future. Blueshift, in turn, results in the **reverse of** a [Time Dilated Past](https://github.com/orgs/a-cosmology-group/discussions/160): clocks shall be running faster with increasing VM.
In the world of gravity this means that the period of an orbital clock is inversely proportional to the square root of the changing mass. Formally, if $T=$ period of the moon's orbit, $M=$ mass of earth, $r=$ earth-moon distance, then $T/T_0=\sqrt{M_0/M}$ in:

$$
m \omega^2 r = G\frac{m M}{r^2} \quad \Longrightarrow \quad r^3 = \frac{G.M}{(2\pi/T)^2}
$$

$$
\left(\frac{r}{r_0}\right)^3 = \frac{M/M_0}{(1/\sqrt{M_0/M})^2} = 1
$$

With other words, with Newtonian mechanics, _there is no change at all in the Moon's orbit_; it does not increase and it does not shrink. More elaborate information can be found in our [Time Dilation](http://www.alternatievewiskunde.nl/cosmology/klokken.htm) webpage.
Question remains: if the orbit remains the same, how then it can be observed to increase? The secret is in [Length Contraction](http://www.alternatievewiskunde.nl/cosmology/lengte.htm). In that webpage it is shown that all of our measuring equipment is shrinking / blueshifted with atomic time. This includes all of our solid rods and the light waves we are emitting. Thus what we **measure** is equivalent with the **expansion** of empty space, between earth and moon eventually.

Thanks for your reply. Hope this helps.

**Note.** In view of the above, the earth's rotation retardation is **not at all** related to the expansion of the lunar orbit. Provided that explanation of both phenomena is **limited** to the Variable Mass Theory. The reason is that $A\ne 1/H_0$ ; the age of the earth is not (half) the age of the universe. However, for both phenomena the VMT turns out to be only **part** of an explanation. It cannot be excluded that some of the remaining part can be explained by e.g. [Tidal acceleration](https://en.wikipedia.org/wiki/Tidal_acceleration). But calculating for that possible common cause is currently beyond my capabilities.

**Reply by ExpEarth** [reply]
There has to be an error in your logic somewhere.  You introduce time dilation in the past and length contraction in the future.  It's like you are doing some version of special relativity or general relativity, but without any of the benefits of those theories!

It's easy to become really confused when all units are changing over time.  For the Earth-Moon system let's just put aside our choice of units.  If the Earth's mass continues to increase over time, the Moon's period will decrease and eventually the Moon will collide with the Earth.  The fireball should be apparent no matter what clocks and rods we are using!

**Reply by HanDeBruijn** [reply]
@ExpEarth.
>It's like you are doing some version of special relativity or general relativity

Yes, the Variable Mass Theory [arises from General Relativity](http://www.alternatievewiskunde.nl/cosmology/narlikar.pdf) by a transformation.
> the Moon's period will decrease and eventually the Moon will collide with the Earth.

That reply cannot be the result of serious study. It's the other way around, as you can easily find in e.g. [_Why is Moon slowly drifting away from Earth?_](https://www.livemint.com/news/world/why-is-moon-slowly-drifting-away-from-earth-11677335970765.html) : _If the Moon continued to drift away from Earth at the same rate, then it would have possibly collided with Earth around 1.5 billion years ago, which never happened in past._ Indeed it has never happened in the past, as explained by my theory.

**Reply by ExpEarth** [reply]
The correct way to look at this in my view is that the Moon and the Earth did have close contact in the past, not 1.5 billion years ago but rather 4.5 billion years ago.  This is when the Moon was formed by a process of fission from the Earth (various mechanisms have been proposed for that and I have my own).  The prevailing theory that the Moon arose by impact of a Mars-size body (Theia) is a load of rubbish.  The Moon went into a low orbit around the Earth and gradually receded from it.  The reasons for the recession are complex and not fully known, but only a part of it can be explained by tidal friction.

This is all different from what you are suggesting.  If the Earth's mass is steadily increasing, then the Moon can't help but slowly spiral down into it.  We have no indication for this happening.

**Reply by HanDeBruijn** [reply]
>This is all different from what you are suggesting.

This is all different from what **you** are suggesting **by distorting** my theory. A well known trick that kills any fair debate. If you want to have the last word, please go ahead.

**Reply by ExpEarth** [reply]
I don't see where I distorted anything.  I'm just pointing out a logical inconsistency.

**Reply by HanDeBruijn** [reply]
Maybe our mutual misunderstanding can be resolved by assuming that @ExpEarth does not distrust General Relativity, more specifically [Hubble's law](https://en.wikipedia.org/wiki/Hubble%27s_law): _The motion of astronomical objects due solely to this [cosmic] expansion is known as the **Hubble flow**. It is described by the equation_ $v=H_0 D$ . Where $D=r_0\approx$ current earth-moon distance and $H_0=$ Hubble parameter. This results in the same outcome as with the Variable Mass Theory. Of course it does, because the latter is a "conformal" transformation of the former: $\Delta r = v.t = H_0 r_0 t\approx 2.89$ centimeters / year .
>The correct way to look at this in my view is that the Moon and the Earth did have close contact in the past, not 1.5 billion years ago but rather 4.5 billion years ago. This is when the Moon was formed by a process of fission from the Earth [ .. ]

It might be claimed that the VMT can calculate with Narlikar's law $(1+H_0\tau/2)^2=m/m_0$ and $\tau=-A=$ where $A=$ age of the event:
```
# 1 megaParsec
Mpc := 3.08567758*10^22;
# Hubble parameter
H_0 := 73.4*1000/Mpc;
# Year in seconds
year := 31556926;
# Earth-Moon distance (m)
r_0 := 384.4*10^6;
# Age of Earth(-Moon system)
A := 4.543*10^9*year;
# Distance at that time
r := r_0*(1-H_0*A/2)^2;
```
Giving $r(A) \approx 264,500$ km, which is not compatible with _a process of fission from the Earth_.
However, we have seen that, in another frame of reference, the distance between earth and moon is invariable. And what is the physical meaning of a result that depends on distance measurement with redshifted light. At that time there were no humans available in place to carry out such experiments.

**Reply by ExpEarth** [reply]
Han, I wonder if you could try putting into words, with no equations at all, why you suppose that the Variable Mass Theory can have a Earth-Moon distance that is either increasing or unchanged.  I can get no qualitative feel whatsoever from your equations.

**Reply by HanDeBruijn** [reply]
@ExpEarth . Modern physics is characterized by relativity and quantum mechanics. The former comes with [Frames of reference](https://en.wikipedia.org/wiki/Frame_of_reference), the latter comes with the [Uncertainty principle](https://en.wikipedia.org/wiki/Uncertainty_principle). Implication of both is that physics as observed, grossly speaking, depends on the observer.
This effect is enlarged by the Variable Mass Theory. And it becomes more and more disturbing with physics on cosmological distances and at cosmological timespans. Matt, to be honest, I can hardly understand my own theory for those far away regions of space and time. 
This is my last sentence in a webpage called [Suport by Hoyle](http://www.alternatievewiskunde.nl/cosmology/Hoyle.htm): _Physics is part of our technology. Nothing of its "truth" can be considered as being independent of that technology, as it has been made up by us humans._ In the end, **We are part of Nature** ourselves, whether we appreciate it or not.

**Reply by ExpEarth** [reply]
Well, all I can say is good luck with your model, Han.  You are obviously deeply attached to it. It's hard for any of us to walk away from our most cherished models.  For me your model seems not to work very well, or at least does not provide explanations simpler than the mainstream ones.  The remote regions of space and time are precisely the ones where your VMT should provide clear, alternative interpretations.  It should be troubling for you that the JWST is showing remote galaxies to be more or less similar to nearby ones.

**Comment by RedshiftDrift** [comment]
"Are atomic clocks slowing down? Possible connection between the quantum and the cosmos: A feasible experimental test of objective realism"
https://arxiv.org/abs/1402.4065

@HanDeBruijn what if it's not the mass that's changing, but instead Planck's constant that is changing?  As suggested in the paper cited above, this would also produce a "redshift".

The paper's experimental proposal is flawed: measuring an atomic line with a clock that uses another atomic line is circular logic.
However, is there a way to tell experimentally which process, if any, is happening: "VM" or "Vh"?

**Reply by HanDeBruijn** [reply]
>what if it's not the mass that's changing, but instead Planck's constant that is changing?

Similar ideas have been coined up by Patrick Tonin (@PatrGT) for example. However, Planck's constant is also part of the (in)famous dimensionless number $\alpha$, the fine-structure constant. It is conjectured by some that $\alpha$ might be pure mathematical, instead of physical, as is pointed out in [this reply](https://github.com/orgs/a-cosmology-group/discussions/276#discussioncomment-10929906). From [Wikipedia](https://en.wikipedia.org/wiki/Rydberg_constant) we have (for hydrogen), formally

$$
Ry = \frac{\alpha^2}{2}m_e c^2 \quad ; \quad \frac{1}{\lambda} = Ry.\frac{1}{hc}\left(\frac{1}{n_1^2}-\frac{1}{n_2^2}\right)
$$

If $\alpha$ is an invariant pure number, then we thus have that redshift is simply proportional to Planck's constant (without Variable Mass and provided that the lightspeed $c$ does not change, as in some other theories). This is all contrary to findings in [the paper](https://arxiv.org/abs/1402.4065)

**Comment by PatrGT** [comment]
> "what if it's not the mass that's changing, but instead Planck's constant that is changing?"
Both mass and Planck's constant could be changing, as well as the electron compton time and the graviational constant. I have proposed that h varies with t-3, electron mass with t-2, electron compton time with t-1, G with t-1 and the mass of the Universe with t2. That obviously implies a continuous matter creation that could be considered to violate the law of conservation of energy. However, this would not be the case if it was counterbalanced by a continuous creation of negative gravitational potential energy. In anycase, the numbers and equations seem to match nicely.   #https://www.researchgate.net/publication/346258792_The_Quantum_Bang_Hypothesis_An_Alternative_to_Dark_Matter_and_Dark_Energy
Do you have any views on this proposal ?

**Reply by HanDeBruijn** [reply]
@PatrGT . I've seen several versions of this article and I wonder what version is the most up to date one.

**Reply by PatrGT** [reply]
Hi @HanDeBruijn , there should be only one version, the one published in 2020 in JHEPGC journal. The link above takes you to it.
